docs/scaling/secrets.md

```diff
@@ -183,3 +189,43 @@ AWS Secrets Manager console) as a source:
 @secrets(sources=['arn:aws:secretsmanager:us-west-2:001234556000:secret:some-secret'])
 ```
 
+### Accessing secrets in Azure
+
+Azure Key Vault is an account specific service, managed via Azure
+Resource Manager. Currently, only `Secret` object types are supported. You can specify 
+secrets in the `sources` list or dictionary object as shown below. 
+
+The following formats of secrets are supported. 
+
+Fully qualified Key Vault Id:
+
+```python
+@secrets(sources=['https://az-key-vault.vault.azure.net/secrets/secretkey/2260d88aca504269999c5f9413c3abcd'])
+```
+
+Key Vault Id without version:
+
+```python
+@secrets(sources=['https://az-key-vault.vault.azure.net/secrets/secretkey'])
+```
+
+Key Vault Object Name with version:
+
+```python
+@secrets(sources=['secretkey/2260d88aca504269999c5f9413c3abcd'])
+```
+
+Key Vault Object Name:
+
+```python
+@secrets(sources=['secretkey'])
+```
+
+:::info
+
+If the Azure Key Vault URL is not specified in the sources attribute, it must be set in 
+the metaflow configuration as:
+
+"METAFLOW_AZURE_KEY_VAULT_PREFIX": "https://az-key-vault.vault.azure.net/"
```

Comment:
not sure if formatting can be applied here, but a code-wrap would be nice

Reply:
![Screenshot 2024-05-20 at 10 25 52 AM](https://github.com/Netflix/metaflow-docs/assets/144727284/2c407309-9364-458b-8e71-2c4b6cdca8b0)


from the vercel preview - this is what it kinda looks like atm. I think the info wrapping is ok given its a metaflow config rather than any code as such

Reply:
@saikonen after adding the code-wrap it looks like this:
<img width="975" alt="Screenshot 2024-05-20 at 6 13 51 PM" src="https://github.com/Netflix/metaflow-docs/assets/144727284/1865666b-7aab-4f91-be69-92effd40fdb6">

Reply:
excellent, looks nice with the wrap highlighting :)